actionview/actionview.gemspec

```diff
@@ -22,6 +22,7 @@ Gem::Specification.new do |s|
   s.add_dependency "activesupport", version
 
   s.add_dependency "builder",       "~> 3.1"
+  s.add_dependency "erubi",         "~> 1.4"
   s.add_dependency "erubis",        "~> 2.7.0"
```

Comment:
Can we drop the Erubis dep? Perhaps by moving the handler to a separate lazy-loaded file with an explicit erubis require or by wrapping the existing impl with a `begin require 'erubis'; rescue LoadError; else <ERubis handler> end`.